proto/storage/compliance_operator_v2.proto

```diff
@@ -85,10 +85,11 @@ message ComplianceOperatorRuleV2 {
   string instructions = 17;
 }
 
-// Next Tag: 3
+// Next Tag: 4
 message RuleControls {
   string standard = 1; // @gotags: search:"Compliance Standard,hidden"
-  repeated string controls = 2;
+  repeated string controls = 2 [deprecated = true];
```

Comment:
This is a good idea.  This way the change won't break staging and we can remove it later when we do the migration.